es-app/src/guis/GuiGamelistOptions.cpp

```diff
@@ -60,13 +77,26 @@ GuiGamelistOptions::GuiGamelistOptions(Window* window, SystemData* system) : Gui
 GuiGamelistOptions::~GuiGamelistOptions()
 {
 	// apply sort
-	FileData* root = getGamelist()->getCursor()->getSystem()->getRootFolder();
-	root->sort(*mListSort->getSelected()); // will also recursively sort children
-
-	// notify that the root folder was sorted
-	getGamelist()->onFileChanged(root, FILE_SORTED);
+	if (!fromPlaceholder) {
+		FileData* root = getGamelist()->getCursor()->getSystem()->getRootFolder();
+		root->sort(*mListSort->getSelected()); // will also recursively sort children
+
+		// notify that the root folder was sorted
+		getGamelist()->onFileChanged(root, FILE_SORTED);
+	} 
+	if (mFiltersChanged) 
+	{
+		ViewController::get()->reloadGameListView(mSystem);
```

Comment:
Actually, I tried a bit more and reduced calling the reloadGameListView to only when we're coming from a placeholder to a full list, reason being that otherwise the first selected element did not show any image, video, metadata, etc. Only after I moved.

Every other filtering event turns into a onFileChanged - thanks for the pointer!

Please test to make sure nothing is fully broken - just tried it briefly this morning, but will only be able to test either later or in the coming days.

Thanks.